.vscode/settings.json

```diff
@@ -0,0 +1,6 @@
+{
+    "files.associations": {
+        "vector": "c",
+        "xutility": "c"
+    }
+}
```

Comment:
Please do not commit IDE support files.

Reply:
I have deleted it. Thanks for your correction. ^v^